ui/src/main/java/com/midtrans/sdk/ui/constants/DateTimeConstant.java

```diff
@@ -0,0 +1,11 @@
+package com.midtrans.sdk.ui.constants;
```

Comment:
We need to make this consistent.

For constants variable we have two options:

- Put it on Constants class as inner class.
- Put each inner class on different class.